pkg/blockatlas/marketdata.go

```diff
@@ -35,24 +35,24 @@ type TickerPrice struct {
 }
 
 type Rate struct {
```

Comment:
Where is it used?

Reply:
it's used by storage, market data, and API package 

<img width="1252" alt="Screen Shot 2019-11-13 at 14 34 31" src="https://user-images.githubusercontent.com/2406457/68788721-bdba9d80-0622-11ea-8a04-420a10ed1f8b.png">

Reply:
Not for api result?

Reply:
no, but we can create a route to retrieve the rate as well, now we are using inside the API package only to apply the currency rate in API result